linkcheck/configuration/confparse.py

```diff
@@ -246,9 +248,8 @@ def check_password_readable(self, section, fields):
                 log.warn(
                     LOG_CHECK,
                     _(
-                        "See http://support.microsoft.com/kb/308419 for"
-                        " more info on setting file permissions."
-                    ),
+                        "See %(url)s for more info on setting file permissions."
+                    ) % {"url": "https://support.microsoft.com/kb/308419"}
                 )
```

Comment:
This needs a corresponding change in po/de.po, where this message is currently translated.

(es.po and fr.po don't translate it so there's nothing to lose)

Reply:
I think that will clash with the GNOME proxy commit which has also updated de.po. I think this PR is worth having (in case anyone is using W3C validation), and it saves a rebase...
I will submit another PR after the GNOME one for this.

Reply:
Please do!

Reply:
Ping?

Reply:
I did try it but have been undecided. It is very sensitive to the environment, I get:

```
 po/de.mo           |  Bin 33015 -> 32536 bytes
 po/de.po           | 1714 ++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------
 po/es.po           | 1617 ++++++++++++++++++++++++++++++++++++++++++-----------------------------------------
 po/fr.po           | 1613 +++++++++++++++++++++++++++++++++++++++++-----------------------------------------
 po/linkchecker.pot | 1287 +++++++++++++++++++++++++++++++++---------------------------------
 5 files changed, 3146 insertions(+), 3085 deletions(-)
```

an awful lot of just moving things around which makes that diff pretty unintelligible. OTOH it might make the next one clearer - if I create it.

 I don't really mind. I'm happy to submit that if you still want it.

Reply:
That is to say, there will need to be an update. The question is just whether to do one now or only after the currently open PRs.

Reply:
What about a simple manual :%s,https://support.microsoft.com/kb/308419,%(url)s, in the pot and po files?  Just so this doesn't get forgotten.

(I've intentionally been staying away from reviewing other PRs because I didn't want to forget about this one.)

Reply:
I've created  #447, to be safe by copying the individual change in the pot file, regenerating the po's and editing in gtranslator, and then generating the mo.